src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -107,6 +107,12 @@ def load_command_table(self, _):
         g.custom_command('set', 'update_connection_strings')
         g.custom_command('delete', 'delete_connection_strings')
 
+    with self.command_group('webapp config azure-storage-accounts') as g:
+        g.custom_command('list', 'get_azure_storage_accounts', exception_handler=empty_on_404)
```

Comment:
Can you double check whether or not `empty_on_404` is needed? If service end doesn't throw, then please remove it.

Reply:
i think we need it